config/routes.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+
+  resources :users do
+    resources :posts
+  end
```

Comment:
Nice job so far👍🏼. just as required for this milestone, kindly ensure your route, only exposes the 4 URL path as required. Apart from this being a requirement, limiting routes to only essential actions can enhance security by reducing the surface area for potential attacks. If certain actions are not needed for a resource, it's best not to expose them.


<img width="796" alt="image" src="https://github.com/kazim110/blog_app/assets/64643995/c1c1e6d2-1664-4d7d-94b0-d5479d6f2ea8">

```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```